source/_integrations/music_assistant.markdown

```diff
@@ -183,19 +203,91 @@ Perform a local search on the Music Assistant library.
+              data:
+                media_id: "{{ random_tracks['items'][repeat.index-1].uri }}"
+                media_type: track
+                enqueue: add
+              target:
+                entity_id: media_player.ma_kitchen_speaker
+```
+
 ### Action `music_assistant.get_queue`
 
-Get the queue details of a Music Assistant player queue.
+Get the queue details of a Music Assistant player queue. This provides programmatic access to comprehensive information about the current and next media item in the queue. This information could be used to create a bespoke media dashboard.
 
 - **Data attribute**: `entity_id`
   - **Optional**: No.
   - **Description**: The entity_id of the player holding the queue to be retrieved.
   - **Example**: `media_player.kitchen_speaker`
 
+#### Example
+
+This example sets the name of the currently playing track in an [`input_text`](https://www.home-assistant.io/integrations/input_text/) which could then be used on a dashboard.
+
+```yaml
+script:
+  get_now_playing:
+    mode: queued
+    alias: "Get Now Playing Track Name"
+    sequence:
+      - action: music_assistant.get_queue
+        data:
+          entity_id: media_player.ma_kitchen_speaker
+        response_variable: queue_info
+      - service: input_text.set_value
+        data:
+          entity_id: input_text.now_playing
+          value: "{{ queue_info['media_player.ma_kitchen_speaker'].current_item.name }}"
+```
+  
 ## Notes
 
 - Any Home Assistant players added to Music Assistant will appear duplicated as the MA version of the player is created. The original HA player can be hidden if desired.
 
+## Known limitations
+
+The `get_queue` action only returns the current and next item in the queue. This is because a large amount of data is returned and if this was done for potentially thousands of tracks this could have an adverse impact on HA performance or stability.
+
+The data returned by the `get_queue` action will be partially limited if the item is not in the library (e.g. an item was selected for playback directly from Spotify). Metadata such as favorite status, explicit status, last played, played count and disc art URL are only available for items which are in the MA library.
+
+Radio mode is only available with certain music providers, and an error will be shown if attempting to enable radio mode on an item that isn't linked to one of those providers. Review the Music Assistant documentation to identify which providers support this functionality.
+
+## Troubleshooting
+
+### Can’t find the MA actions
+
+#### Symptom: No Music Assistant actions are shown in the editor
+
+When trying to set up a script or automation via the GUI, no MA actions can be found.
+
+##### Description
+
+This means the add-on may have been installed, but the integration has not.
+
+##### Resolution
+
+Go to the [Configuration section](https://www.home-assistant.io/integrations/music_assistant/#configuration) and install the integration.
+
```

Comment:
isn't this a bit obvious when looking at it from the integration docs ?

Reply:
I know but the quality level requirement said to include troubleshooting and this is all I could think of! Happy for a suggestion!

Reply:
Discussed and resolved offline